### PR TITLE
Typofixes in console messages

### PR DESCRIPTION
**Pull request by koral--**

### CODE REVIEW

File: analyzer/analyzer.go [hunk 1]
```diff
@@ -274,15 +274,15 @@ if (plugins.hasPlugin('com.android.application')) {
 	if strings.HasPrefix(compileSDKVersion, "android-") {
 		_, err = version.NewVersion(strings.TrimPrefix(compileSDKVersion, "android-"))
 		if err != nil {
-			return "", "", fmt.Errorf("failed to parse compileSDKVersion (%s), error: %s", compileSDKVersion, err)
+			return "", "", fmt.Errorf("Failed to parse compileSDKVersion (%s), error: %s", compileSDKVersion, err)
```
Comment (godrei) [inline analyzer/analyzer.go]: please do not return with capitalised error. The caller function should handle to print the error message with the correct capital.

In this case `analyzeWithGradlew` function is used only in `NewProjectDependencies` function, `NewProjectDependencies` function is called in the `main` function and the error message is wrapped with a capitalised message.

File: analyzer/analyzer.go [hunk 2]
```diff
@@ -291,22 +291,22 @@ if (plugins.hasPlugin('com.android.application')) {
 func parseBuildGradle(buildGradleContent string) (ProjectDependenciesModel, error) {
 	platformVersion, err := parsePlatformVersion(buildGradleContent)
 	if err != nil {
-		return ProjectDependenciesModel{}, fmt.Errorf("failed to determine compileSDKVersion, error: %s", err)
```
Comment (godrei) [inline analyzer/analyzer.go]: please do not return with capitalised error. The main caller function should handle to print the error message with the correct capital.

**Comment by godrei** [comment]
Hi @koral-- ,
thanks for your PR!

I attached some notes to your PR, please have a look and fix them, or let me know if you can not agree with my requests.

**Comment by koral--** [comment]
OK, I will remove capitalization.

**Comment by godrei** [comment]
@koral-- ,
thanks for the PR!

**Pull request merged**